### PR TITLE
Add misc translations [LEI-355]

### PR DESCRIPTION
**Pull request by sheriefvt**
Refs: https://openscience.atlassian.net/browse/LEI-355

## Purpose
Add misc translations including Italian, polish, Thai, Slovenian, Macedonian, and Ukrainian

## Summary of changes
- Download the CSV file for the translation
- Run scripts 
- Generate new / update the locale per each language.

## Testing notes
Use format_translations.py with "--validate" option to verify the correctness of the files.

### CODE REVIEW

File: app/locales/it/config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Ember-I18n includes configuration for common locales. Most users
+// can safely delete this file. Use it if you need to override behavior
+// for a locale or define behavior for a locale that Ember-I18n
+// doesn't know about.
```
Comment (abought) [inline app/locales/it/config.js]: It appears that ember-i18n comes with default settings for italian, so this file can be removed. The config file is only needed for languages that we do not already know how to handle.
https://github.com/jamesarosen/ember-i18n/tree/master/addon/config

File: app/locales/pl/config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Ember-I18n includes configuration for common locales. Most users
```
Comment (abought) [inline app/locales/pl/config.js]: It appears that ember-i18n comes with polish support by default, so this file can be removed.

**Review by abought — COMMENTED** [review]
Looks good!

I have two small requests for the AM. If you think they would take > 1 hr, we can move them to a separate PR.

1. Please delete the two files indicated as likely unnecessary and verify that you see no console messages about missing locale configuration.

2. I notice you were working on improvements to the `consent_form_json.py` script (and README docs). Are those process improvements ready to be included? If you need more time that's fine; we can move these to a separate ticket.

File: app/locales/it/config.js (hunk 1 above)
Comment (sheriefvt) [reply app/locales/it/config.js]: ok thanks will do AM

**Pull request merged**